ml-agents/mlagents/tf_utils/tf.py

```diff
@@ -1,3 +1,10 @@
+from __future__ import absolute_import
```

Comment:
This _is_ the future, you shouldn't need these.

Reply:
embarrassing. fixed

Reply:
The future is now